docs/ui/components/ContentSpotlight/index.tsx

```diff
@@ -19,7 +19,7 @@ type ContentSpotlightProps = {
   caption?: string;
   controls?: any;
   loop?: boolean;
-  maxWidth?: number;
+  maxWidthClass?: string;
```

Comment:
```suggestion
  className?: string;
```

If possible, we should use generic `className` prop, it might be confusing a bit, since in this case the class only applied to Images not Video component now, but maybe something that we should change.

If we want to use the custom class prop it need to be added to the possible props name to scan for ESLint Tailwind plugin:
* https://github.com/expo/expo/blob/main/docs/.eslintrc.cjs#L33